apps/estuary/config/integration.exs

```diff
@@ -9,6 +9,7 @@ config :prestige,
   ]
 
 config :estuary,
+  elsa_endpoint: [localhost: 9092],
   divo: "docker-compose.yml",
```

Comment:
Is this still correct? Is this what it is on aws?

Reply:
This is the integration config